README.md

```diff
@@ -11,7 +11,11 @@ Install
 Usage
 ------------------------
 
-Import into a project file: `const Device = require('react-native-device-detection');`
+Import into a project file: 
+
+Require Import: `const Device = require('react-native-device-detection');`
+
+ES6 Modules: `import { isAndroid } from 'react-native-device-detection`;
```

Comment:
Does this actually work?

Reply:
@m0ngr31 - Indeed it does. Here is an example from our Production Application. I copied and pasted from our Main.js

```import { isAndroid, isTablet } from 'react-native-device-detection';```

Reply:
Awesome